### ISSUE TITLE
Endpoints returning 404

### ISSUE THREAD

**Issue by C-onnor** [airesvsg/acf-to-rest-api#117]
In Postman for Google chrome I tried the following
http://localhost/wp-json/acf/v3/posts

which returned:
```JSON
{
  "code": "rest_no_route",
  "message": "No route was found matching the URL and request method",
  "data": {
    "status": 404
  }
}
```

Why is it returning this with the plugin and acf both enabled?

**Comment by airesvsg** [comment]
Hello @C-onnor,
Thanks for using my plugin.
You set the request API version?
https://github.com/airesvsg/acf-to-rest-api#request-api-version
Thanks

**Comment by C-onnor** [comment]
![no settings](https://cloud.githubusercontent.com/assets/5251892/24072884/283af6e8-0bc4-11e7-80d1-e0c5ec30be3a.PNG)
I don't have the "Settings" text on the plugins page as specified in the docs (picture of what I see above).  I have also tried the following lines in my wp-config.php:
`define( 'ACF_TO_REST_API_REQUEST_VERSION', 3 );`

Also tried this in my child theme's functions.php:
` // Enable the option show in rest
add_filter( 'acf/rest_api/field_settings/show_in_rest', '__return_true' );

// Enable the option edit in rest
add_filter( 'acf/rest_api/field_settings/edit_in_rest', '__return_true' );`

**Comment by airesvsg** [comment]
Hi @C-onnor,
you aren't using the v3. Follow the steps below to download the new version.

1. Download ( https://github.com/airesvsg/acf-to-rest-api/archive/master.zip ) and extract the v3 files;
2. Copy the acf-to-rest-api-master folder into your wp-content/plugins folder;
3. Rename acf-to-rest-api-master to acf-to-rest-api;
4. Activate the ACF to REST API plugin via the plugin admin page.

Thanks

**Comment by C-onnor** [comment]
Thank you for the help don't know how I overlooked that.

**Comment by lance-anderson** [comment]
I ended up down this same rabbit hole. Thankful to find this issue. I ended up in this state b/c I downloaded v2 from WP plugins: https://wordpress.org/plugins/acf-to-rest-api/. I hadn't noticed the v2 installation vs v3 documentation.

**Comment by PythonHe** [comment]
Hey @airesvsg ,

Sorry to ask you about this question again, I am using v3 of ACF to REST API and I did every step you listed above, but I am still getting the `rest_no_route" error`. I've also tried `define( 'ACF_TO_REST_API_REQUEST_VERSION', 3 );` this as well, no luck :( . 

I am currently using ACF Pro version and I include the ACF as part of the theme. 

Any sugguestions will be appreciated! 

Thanks in advance.
